README.md

```diff
@@ -83,7 +83,8 @@ We accept but don't fully support the following types of images:
 Our `export` method provides support for the following options:
 
 - options `Object`
-    - quality `Number` - Quality of output image on a scale from 0 to 100
+    - format `String` - Output image format to callback with. Currently, png and jpeg are available.
+    - quality `Number` - If you are outputting a jpeg, the quality can be specified from 0 to 100.
```

Comment:
Unlike `phantomjssmith`, we support `quality` for any type (only `phantomjssmith's` JPEG library supports it). Although, I have no idea if ImageMagick/Graphics Magick do anything with `quality` for PNGs